packages/central-server/src/social/feedScraper.js

```diff
@@ -43,28 +43,19 @@ const addLatestSurveyFeedItems = async models => {
     ? latestSurveyFeedItem.creation_date
     : minimumDateOfSurveyToAddToFeed;
 
-  const publicPermissionGroup = await models.permissionGroup.findOne({ name: 'Public' });
-  const publicLevelSurveys = await models.survey.find({
-    permission_group_id: publicPermissionGroup.id,
-  });
-  const newSurveys = await getLatestSurveyResponses(models, lastSurveyFeedItemDate, {
-    survey_id: publicLevelSurveys.map(survey => survey.id),
-  });
+  const newSurveyResponses = await getLatestSurveyResponses(models, lastSurveyFeedItemDate);
 
   // Use async for loop instead of Promise.all because this process doesn't block
   // user events and should pace itself to avoid overloading the database.
-  for (let i = 0; i < newSurveys.length; i++) {
+  for (let i = 0; i < newSurveyResponses.length; i++) {
```

Comment:
Is there a stylistic/technical reason for using an indexed `for` here over `for (surveyResponse of newSurveyResponses) {}`?

Reply:
Not really - I didn't write this particular part, just updated it to include all survey responses